library/L2_HAL/actuators/servo/servo.hpp

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+#include <cstdint>
+#include <cstdio>
+
+#include "L1_Peripheral/pwm.hpp"
+#include "utility/map.hpp"
+
+namespace sjsu
+{
+class Servo
+{
+ public:
+    static constexpr uint32_t kDefaultFrequency = 50;
+    static constexpr float kMinAngle = 0;
+    static constexpr float kMaxAngle = 360;
+    static constexpr float kMinPulse = 500;
+    static constexpr float kMaxPulse = 2500;

```

Comment:
```suggestion
    static constexpr float kDefaultMaxPulse = 2000;
```
Name all of these Default. Also, use the most sane values for these. For example, 0 to 90 deg,(support for old servos), and 1000uS to 2000uS (also support for old servos).